content/telegraf/v1.21/administration/enterprise-plugins.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
I'm guessing we can probably just remove this doc. We can either do it in this PR or with Telegraf 1.22.0